sim/firesim-lib/src/main/cc/fesvr/firesim_fesvr.h

```diff
@@ -2,89 +2,50 @@
 #ifndef __FIRESIM_FESVR_H
 #define __FIRESIM_FESVR_H
 
-#include <fesvr/htif.h>
-#include <string>
-#include <vector>
-#include <deque>
-#include <stdint.h>
-#include "midas_context.h"
+#include "testchip_fesvr.h"
 
-#define FESVR_CMD_READ 0
-#define FESVR_CMD_WRITE 1
-
-#define FESVR_ADDR_CHUNKS 2
-#define FESVR_LEN_CHUNKS 2
-
-// Wraps read/write requests that will be shortcircuited through the LOADMEM widget
-struct fesvr_loadmem_t {
-    fesvr_loadmem_t(): addr(0), size(0) { }
-    fesvr_loadmem_t(size_t addr, size_t size): addr(addr), size(size) { }
-    size_t addr;
-    size_t size;
+struct firesim_loadmem_t {
```

Comment:
Nit: Is there not just a standard c++ tuple construct? Maybe just typedef that instead of creating your own version?

Reply:
This is just the old `fesvr_loadmem_t` renamed. I don't really want to change how this works here.